heron/common/src/cpp/network/connection.cpp

```diff
@@ -75,12 +75,10 @@ Connection::~Connection() {
   delete[] mIOVector;
 }
 
-sp_int32 Connection::sendPacket(OutgoingPacket* packet) { return sendPacket(packet, NULL); }
```

Comment:
It is unused, I don't know why an unused parameter contributes to generality. Mind to be more specific?
Again, we can always add it back if we need it in the future, it is pretty easy to do with git.